src/TodoListItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+    return (
+        <span>
```

Comment:
I like keeping the html as simple as possible by only using html tags that are needed! I think it makes the code easier to maintain over time. Would you mind removing the span element?